package.json

```diff
@@ -14,11 +14,12 @@
     "@emotion/react": "^11.13.3",
     "@emotion/styled": "^11.13.0",
     "@mui/material": "^6.1.4",
-    "@stoplight/json-schema-viewer": "^4.16.1",
+    "@observablehq/plot": "^0.6.17",
+    "@stoplight/json-schema-viewer": "^4.16.4",
```

Comment:
There's a number of console deprecation warnings which I hoped might be cleared up my a version bump, but alas no luck! There's already outstanding issues for them in the @stoplight/json-schema-viewer repo, so hopefully cleared up in a future release :crossed_fingers: